### DISCUSSION TITLE
Remove SVT-HEVC?

### DISCUSSION THREAD

**Discussion by 1480c1** [m-ab-s/media-autobuild_suite#2783] [Polls]
<https://github.com/OpenVisualCloud/SVT-HEVC>

Since SVT-HEVC is now marked as archive only, and will no longer get any more updates, I'm thinking about removing it, especially as it has not seen any real updates or maintenance to it for a while.

**Comment by classicdoomenjoyer** [comment]
I'd recommend you to keep it as it is and even tho I don't expect any major updates, but, please keep updating / maintaning the patches, so the people will be able to use the encoder. Its really an efficient encoder of HEVC.
